.travis.yml

```diff
@@ -29,8 +29,10 @@ before_script:
     - npm install
 
 script:
-    - rm -f *sqlite
+    #- rm -f *sqlite
     - cargo build # Linter is also executed here
     - $TRAVIS_BUILD_DIR/tools/execute-unit-tests-with-coverage
-    - jshint static/main/js/*.js static/setup/js/*.js test/selenium/*.js
-    - (cargo run -- -l $BOX_LOCAL_NAME -p $BOX_PORT &> /dev/null &) ; npm test
+    - jshint static/main/js/*.js static/setup/js/*.js test/selenium/*.js test/integration/lib/*.js test/integration/test/*.js
+    - npm run test-integration # starts foxbox and kills it within the test script
+    - rm /home/travis/.local/share/foxbox/users_db.sqlite
```

Comment:
As the path is not that trivial anymore, I'd the removal process in a script (a npm one should be fine) and call that script here.

For instance:
```sh
npm run clean
```